### ISSUE TITLE
[Bug] Upgraded Dark Tank drops 16,000mb version

### ISSUE THREAD

**Issue by Darkfiend009** [rubensworks/EvilCraft#908]
#### Issue type:

- :bug: Bug 

____

#### Short description:

Dark Tank is not dropping upgraded versions when removed from the world.

#### Steps to reproduce the problem:

1. Craft Dark Tank
2. Upgrade Dark Tank in crafting grid by combining Dark Tanks
3. Place Dark Tank in the world
4. Break Dark Tank
5. Reverts to 16,000mb version.

#### Expected behaviour
Dark Tank to keep upgraded capacity when broken.
____

#### Versions:

- This mod: 1.18.1-1.1.16
- Minecraft: 1.18.1
- Forge: 39.1.2

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
Seems to be working fine on my end. I see you didn't mention this mod's version. Could you try again with the latest?
If you're using other mods in your pack, it may also be caused by some other mod.

**Comment by Darkfiend009** [comment]
Sorry, i had the mod version under forge for some reason. I edited the original post with the proper versions.

I just tested with only Cyclops, Evilcraft, and Common Capabilities and it is downgrading the tank after breaking it.

Steps Taken:

1. Craft Dark Tank
2. Upgrade Dark Tank in crafting grid by combining Dark Tanks
3. Place Dark Tank in the world
4. Break Dark Tank
5. Reverts to 16,000mb version.


![image](https://user-images.githubusercontent.com/42146629/161822781-4dd1c8dc-9380-4552-93fd-b8ea97a117f9.png)
![image](https://user-images.githubusercontent.com/42146629/161822850-7a03e3a5-78db-4f46-ac8b-f413f5c8244b.png)
![image](https://user-images.githubusercontent.com/42146629/161822911-1f93743f-8c2e-4295-811c-f7e1f8809d93.png)
